src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx

```diff
@@ -0,0 +1,121 @@
+## Change Default Log Level
+
+Below is an example of setting the default log level to `WARN`:
+
+import iosDefaultLogLevel from '/src/fragments/lib/logging/ios/change_log_levels/10_default_log_level.mdx';
+import androidDefaultLogLevel from '/src/fragments/lib/logging/android/change_log_levels/10_default_log_level.mdx';
+import commonDefaultLogLevel from '/src/fragments/lib/logging/common/change_log_levels/10_default_log_level.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonDefaultLogLevel}}/>
+    </Block>
+    <Block name="Without Configuration File">
```

Comment:
can we rename this into something a bit more indicative? like "configuration in code" or something like that? Maybe you can help me come up with a better name